source/uri-options/uri-options.rst

```diff
@@ -3,15 +3,15 @@ URI Options Specification
 =========================
 
 :Spec Title: URI Options Specification
-:Spec Version: 1.7.0
+:Spec Version: 1.8.0
 :Author: Sam Rossi
 :Spec Lead: Bernie Hackett
 :Advisory Group: Scott L'Hommedieu
 :Approver(s): Cailin Nelson, Jeff Yemin, Matt Broadstone, Dan Pasette, Prashant Mital, Spencer Jackson
 :Informed: drivers@
 :Status: Accepted (Could be Draft, Accepted, Rejected, Final, or Replaced)
 :Type: Standards
-:Last Modified: 2021-4-15
+:Last Modified: 2021-9-xx
```

Comment:
Needs correct date.

Reply:
I'll write in the current date when I'm about to merge; that way the last modified date is in fact accurate.

Reply:
@benjirewis Don't forget the leading zero for "09" when you do. I'm not sure why some of the specs omit that, but I think leading zeroes make sense for YYYYMMDD date formats.